### PR TITLE
Add `databricks-2022-10-01-preview`

### PR DESCRIPTION
**Pull request by favoretti**

### CODE REVIEW

File: swagger
Comment (tombuildsstuff) [inline swagger:1]: can you revert the change to the submodule?

Comment (favoretti) [reply swagger:1]: Sorry about that, multitasking.

**Review by stephybun — APPROVED** [review]
LGTM 👍
